server/views/partials/reviewPanelRadios.njk

```diff
@@ -26,9 +25,12 @@
                value: "expired",
                text: "More than 48 hours have elapsed since the incident"
              }
-          ] | toSelect('value', 'text'),
+          ] | toSelect('value', 'text', pageData.reason),
 					classes: '',
-    			errorMessage: 'rejectedReasonId' | findError(pageData.errors)
+          attributes: {
```

Comment:
This formatting is not good - we really have to review this formatter you are using.

For one, why is it using tabs? Also, is this a good way of formatting - to have newlines for single attributes?

Reply:
that formatting is github, its fine in the code